.travis.yml

```diff
@@ -4,10 +4,17 @@ git:
 env:
   global:
     - ATOM_ACCESS_TOKEN=da809a6077bb1b0aa7c5623f7b2d5f1fec2faae4
-    
-  matrix:
     - NODE_VERSION=0.12
 
+matrix:
+  include:
+    - os: osx
+      env: ATOM_SPECS_TASK=core
+    - os: osx
+      env: ATOM_SPECS_TASK=packages
+  exclude:
```

Comment:
It does spawn unnecessary builds - I should be able to just remove the `osx` entry below though.